RecoTracker/IterativeTracking/python/MixedTripletStep_cff.py

```diff
@@ -337,14 +341,22 @@
 mixedTripletStep = ClassifierMerger.clone()
 mixedTripletStep.inputClassifiers=['mixedTripletStepClassifier1','mixedTripletStepClassifier2']
 
-#LWTNN selector
+trackingPhase1.toReplaceWith(mixedTripletStep, mixedTripletStepClassifier1.clone(
+	mva = dict(GBRForestLabel = 'MVASelectorMixedTripletStep_Phase1'),
+	src = 'mixedTripletStepTracks',
+	qualityCuts = [-0.5,0.0,0.5]
+))
+(trackingPhase1 & fastSim).toModify(mixedTripletStep,vertices = "firstStepPrimaryVerticesBeforeMixing")
```

Comment:
this is not needed and may be a bit confusing re pp_on_AA and highBeta below: the fastsim value is inherited from L332 and this line can be removed

Reply:
Unnecessary line removed